dask/tests/test_threaded.py

```diff
@@ -147,7 +144,7 @@ def test_interrupt():
             proc.send_signal(sigint)
             # Wait a bit for it to die
             start = time()
-            while time() - start < 2.0:
+            while time() - start < 4.0:
```

Comment:
Yeah. I spent some time debugging on travis, and couldn't seem to reproduce (editing our test suite to just run this test a bunch). A bit baffled by this periodic failure.